appveyor.yml

```diff
@@ -12,6 +12,7 @@ environment:
   - ruby_version: 23-x64
   - ruby_version: 24-x64
   - ruby_version: 25-x64
+  - ruby_version: 26-x64
```

Comment:
ruby didn't seem to get installed?

Reply:
@segiddins

With normal CI, all Ruby builds (except trunk) used on Appveyor are pre-installed.

1. The RubyInstaller2 2.6.0 build is not passing due to a last minute update to BigDecimal, which changed spec tests (added to ruby/ruby, not yet added to ruby/spec).

2. Once released, Appveyor may take a few days (or weeks) to add the build to their images.

3. Trunk/head is not affected.

4. I'll ping when it's been added...